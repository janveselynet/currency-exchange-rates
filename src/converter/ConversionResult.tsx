import React from 'react';

interface ConversionResultProps {
  amountInCzkToConvert: number;
  amountInTargetedCurrency: number;
  targetedCurrencyCode: string;
};

export default function ConversionResult({
  amountInCzkToConvert,
  amountInTargetedCurrency,
  targetedCurrencyCode
}: ConversionResultProps): JSX.Element {
  return <p>{amountInCzkToConvert} CZK = {amountInTargetedCurrency} {targetedCurrencyCode}</p>;
}
