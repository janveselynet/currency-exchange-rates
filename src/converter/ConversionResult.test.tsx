import { render } from '@testing-library/react';
import React from 'react';
import ConversionResult from './ConversionResult';

describe('ConversionResult', () => {
  it('renders properly', () => {
    const props = {
      amountInCzkToConvert: 100,
      amountInTargetedCurrency: 250,
      targetedCurrencyCode: 'USD',
    };
    const { container } = render(<ConversionResult {...props} />);
    expect(container).toMatchSnapshot();
  });
});
