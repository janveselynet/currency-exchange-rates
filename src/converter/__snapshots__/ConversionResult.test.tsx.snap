// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConversionResult renders properly 1`] = `
<div>
  <p>
    100
     CZK = 
    250
     
    USD
  </p>
</div>
`;
